src/render/Renderer.cpp

```diff
@@ -1515,6 +1515,10 @@ bool CHyprRenderer::applyMonitorRule(CMonitor* pMonitor, SMonitorRule* pMonitorR
         force = true;
     }
 
+    if (pMonitorRule->drmMode.type == DRM_MODE_TYPE_USERDEF) {
```

Comment:
why? Just store the last modeline

Reply:
I didn't want to touch the scary monster if-condition right below. I thought it not important because re-applying the same mode doesn't seem to do anything anyway (that I know of). What do you think of simply removing that whole already-applied check for simplicity?

Reply:
it's there for a reason, it sets frame skips, monitor reloads, layer reloads, damage and stuff. The "monster if" is not that scary... store the modeline in a string and compare it